### PR TITLE
Design Setting page

### LINKED ISSUE #15: Design Setting page component

### PR DESCRIPTION
**Pull request by anthonyminyungi**
- It closes #15

### CODE REVIEW

File: src/components/common/button/index.tsx [hunk 1]
```diff
@@ -18,16 +18,19 @@ const useStyles = makeStyles(() => ({
 type ButtonProps = {
     children: string,
     icon?: JSX.Element,
+    onClick?: (event: React.MouseEvent<{}>) => void,
 }
 
-export default function BaseButton ({ children, icon }: ButtonProps) {
+export default function BaseButton ({ children, icon, onClick }: ButtonProps) {
```
Comment (anthonyminyungi) [inline src/components/common/button/index.tsx:24]: Added function prop to base button.

File: src/components/login/index.tsx [hunk 2]
```diff
@@ -51,8 +61,9 @@ export default function LoginPage() {
                 <TextField className={classes.input} type="password" label="Password" />
                 <BaseButton icon={<LockOpenIcon />}>
                     Start 0auth
-                </BaseButton> 
+                </BaseButton>
             </form>
+            <Button className={classes.setPass}>Set your password</Button>
```
Comment (anthonyminyungi) [inline src/components/login/index.tsx:66]: Appearance.
스크린샷 2020-09-02 오후 12 55 14

File: src/components/setting/index.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                    Password setting
+                </Typography>
+                <TextField className={classes.input} type="password" label="Password" />
+                <BaseButton>
+                    Save Password
+                </BaseButton>
+            </Container>
+        </Layout>
+    );
+};
```
Comment (anthonyminyungi) [inline src/components/setting/index.tsx:52]: Appearance.
스크린샷 2020-09-02 오후 12 55 40

**Review by anthonyminyungi — COMMENTED** [review]
Self-review.

File: src/components/login/index.tsx (hunk 2 above)
Comment (HyeockJinKim) [reply src/components/login/index.tsx:66]: Cool!

**Review by HyeockJinKim — APPROVED** [review]
LGTM.

**Pull request merged**